extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -860,6 +886,48 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
     this.view.myPubkeyModule.reevaluateShouldAttachOrNot();
   }
 
+  private publicKeysToRenderedText = (pubKeyInfos: PubKeyInfo[]): string => {
+    const valid: PubKeyInfo[] = [];
+    const expired: PubKeyInfo[] = [];
+    const revoked: PubKeyInfo[] = [];
+    for (const pubKeyInfo of pubKeyInfos) {
+      if (pubKeyInfo.revoked) {
+        revoked.push(pubKeyInfo);
+      } else if (KeyUtil.expired(pubKeyInfo.pubkey)) {
+        expired.push(pubKeyInfo);
+      } else {
+        valid.push(pubKeyInfo);
+      }
+    }
+
```

Comment:
@IvanPizhenko you see `linter` complains about these empty lines.
You should either remove them, or place some remarks there